.gitignore

```diff
@@ -53,3 +53,4 @@ dkms.conf
 
 .vscode
 game
+/build*/
```

Comment:
What is the `*`for? Is it for the CMake output based on platform?

Reply:
It's there for if you want to generate build script files f.x. clang and visual studio on the same machine. Then you can do generate something like `build.clang` for the clang compiler, and `build.vstudio` for the visual studio compiler.
It also just matches `build` if you don't bother with multiple toolchains.
All of these paths are local to the root .gitignore file (that's what the prefix / does)

Reply:
Wouldn't `/build.*/` be a better fit?

Reply:
Then it wont fit just `build/` ... Anyway, I can make it into `/build/` and then just use sub folders within that, to build for different compilers. Is that an OK solution?

Reply:
So `/build/clang` for instance? That's perfectly fine.

Reply:
Yes. Alright, I'll fix it